test/integration_tests/conftest.py

```diff
@@ -53,6 +62,29 @@ def sample_speech(tmp_path, lang):
     return path
 
 
+@pytest.fixture
+def mixture_source(tmp_path, task):
```

Comment:
Please get rid of `task` parameterization as there is only one at the moment. We can add it when it becomes absolutely necessary.